src/lib/shared/button/SaveInGymlesButton.svelte

```diff
@@ -8,11 +10,15 @@
 	export let slug: string;
 
 	async function downloadGame() {
-		userStore.saveGame(slug).then((success) => dispatcher('saved', { success }));
+		if (navigator.onLine)
```

Comment:
Kan dit geen utils worden? Wordt ook al in andere button gebruikt

Reply:
Ja, maar laten we dat als losse issue doen!

Reply:
Is #301